.github/workflows/build-release.yml

```diff
@@ -0,0 +1,60 @@
+name: Build Release
+on:
+  push:
+    tags:
+    - v*
+jobs:
+  create-release:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - name: Create Release
+      id: create-release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        draft: false
+        prerelease: false
+    - shell: bash
+      run: |
+        echo '${{ steps.create-release.outputs.upload_url }}' > upload-url.txt
+    - name: Save GitHub release upload URL for next job
+      uses: actions/upload-artifact@v1
+      with:
+        name: upload-url
+        path: upload-url.txt
+  build:
+    runs-on: 'macos-latest'
```

Comment:
This virtual machine runs on macOS 10.15 (Catalina). The current binaries are built on macOS 10.13 (High Sierra). We need to check whether the binaries produced by GitHub Actions are backwards compatible.

Reply:
Yes, sadly I think this might be a deal-breaker for the use of GitHub Actions as it is currently provided (and this is why e.g. Homebrew provides prebuilt binaries for each minor/codenamed version of OS X).

Reply:
That would be a bummer. 🤔

Feel free closing this pull requests if it doesn't produce usable artifacts.

Reply:
Perhaps setting the `MACOSX_DEPLOYMENT_TARGET=10.x` environment variable might work:

https://clang.llvm.org/docs/CommandGuide/clang.html#environment